jwst/regtest/test_nirspec_brightobj.py

```diff
@@ -49,3 +52,31 @@ def test_nirspec_brightobj_spec2(run_tso_spec2_pipeline, fitsdiff_default_kwargs
     # Compare the results
     diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
     assert diff.identical, diff.report()
+
+
+@pytest.mark.bigdata
+def test_flat_field_step_user_supplied_flat(jail, rtdata_module, fitsdiff_default_kwargs):
```

Comment:
Since this test doesn't share any data with the other tests in this module, it should use the `rtdata` fixture which is function-scoped.  Data for this test will then be segregated from data from the other tests in this module.